app/src/main/res/values/strings.xml

```diff
@@ -132,6 +132,11 @@
     <string name="dialog_date_info_title_course">Course duration</string>
     <string name="dialog_date_info_title_section">Start date</string>
 
+    <string name="dialog_external_content_title">External content</string>
+    <string name="dialog_external_content_message">This task is opened with an external tool that is not part of the openHPI app. It may not be optimized for mobile devices, so we recommend that you complete it on a computer.\n\nDo you want to open it anyway?</string>
+    <string name="dialog_external_content_yes">Open</string>
+    <string name="dialog_external_content_yes_always">Open and don\'t ask again</string>
```

Comment:
We use `launch` as the button action for LTIs and `open` for Peer Assessments. Let's make this consistent for the dialog options.